frontend/api/client.js

```
@@ -0,0 +1,98 @@
+      accessToken = access_token || accessToken;
+      refreshToken = refresh_token || refreshToken;
+      // Notify listener (AuthContext) so it can persist & update state
+      if (tokenUpdateListener) {
+        tokenUpdateListener({ accessToken, refreshToken });
+      }
+      return accessToken;
+    } finally {
+      isRefreshing = false;
+      const p = refreshPromise; // preserve for awaiting callers
+      refreshPromise = null;
+      return p; // not used
```

Biome (2.1.2)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)